.github/workflows/integration-tests-non-root.yaml

```diff
@@ -2,33 +2,37 @@ name: Integration Tests (Non-root)
 run-name: Integration Tests (Non-root) - ${{ github.run_id }} - @${{ github.actor }}
 
 on:
-  push:
-    branches:
-      - master
-      - branch/*
```

Comment:
Slightly unrelated change - I don't know if we intended to run the integration tests on push, but I don't think we need to be. If it merged, that means it passed the PR and merge queue checks. Plus, if it does fail after push, nobody is paying attention to the failure anyway.

Reply:
Yeah -- I don't see a reason to run these after merge either.